Global.props

```diff
@@ -109,14 +109,14 @@
       Update for every public release. 
     -->
     <SemanticVersionMajor>1</SemanticVersionMajor>
-    <SemanticVersionMinor>2</SemanticVersionMinor>
+    <SemanticVersionMinor>3</SemanticVersionMinor>
```

Comment:
We will definitely need to update the service to pick the latest version. 1.2.0 was the latest stable version that is already shipped and we cannot ship new nuget package with the same version.